cli_tools/common/image/importer/daisy_inflater.go

```diff
@@ -195,16 +195,17 @@ func calculateScratchDiskSize(metadata imagefile.Metadata) int64 {
 	// This uses the historic padding calculation from import_image.sh: add ten percent,
 	// and round up.
 	padded := int64(float64(metadata.PhysicalSizeGB)*1.1) + 1
-	if padded < defaultInflationDiskSizeGB {
-		return defaultInflationDiskSizeGB
-	}
-	return padded
+	return getAllocatedDiskSize(padded)
 }
 
 // Ensure a minimum of 10GB (the minimum size of a GCP disk)
 func calculateInflatedSize(metadata imagefile.Metadata) int64 {
-	if metadata.VirtualSizeGB < defaultInflationDiskSizeGB {
+	return getAllocatedDiskSize(metadata.VirtualSizeGB)
+}
+
+func getAllocatedDiskSize(size int64) int64 {
```

Comment:
```suggestion
func enforceMinimumDiskSize(size int64) int64 {
```

Or something like that to differentiate between this method and  `calculateInflatedSize`

Reply:
thanks, done